src/mscorlib/shared/System/Text/DecoderFallback.cs

```diff
@@ -75,6 +51,14 @@ public static DecoderFallback ExceptionFallback
         // Maximum number of characters that this instance of this fallback could return
 
         public abstract int MaxCharCount { get; }
+
+        internal bool IsMicrosoftBestFitFallback
+        {
+            get
+            {
+                return bIsMicrosoftBestFitFallback;
```

Comment:
It was not automatically renamed because of it was internal. 

CoreLib has friends visibility. CodeFormatter does not rename internal fields in assemblies with friends visibility. It treats them as the assembly public contract.